src/marten/http/session/store/base.cr

```diff
@@ -9,12 +9,27 @@ module Marten
           @accessed : Bool = false
           @modified : Bool = false
           @session_hash : SessionHash? = nil
+          @expires : Int32? = nil
 
           getter session_key
 
           def initialize(@session_key : String?)
           end
 
+          def expires=(value : Int32)
+            @expires = value
+            @modified = true
+          end
```

Comment:
I would prefer that we persist a `Time` object instead of an integer. Eventually, if we want to make it possible to configure that the session expires in a specific amount of times, then I think we should support an additional `#expires_in` method that would accept a `Time::Span` object as input and that would update the `@expires_at` time instance variable accordingly.

Reply:
Ok. Just thought that for a store like Redis, an additional value conversion operation would be required (the TTL is specified as Int there)